alcs-api/src/code/code.service.spec.ts

```diff
@@ -144,7 +140,7 @@ describe('CodeService', () => {
     const mockStatus = {
       uuid: '2',
     };
-    mockStatusRepository.findOne.mockResolvedValue(mockStatus as CardStatus);
+    mockStatusRepository.findOne.mockResolvedValue(mockStatus as any);
```

Comment:
the same with any

Reply:
Changed back to as CardStatus and fixed.